contextualbandits/utils.py

```diff
@@ -285,7 +285,7 @@ def _check_autograd_supported(base_algorithm):
             raise ValueError("Automatic gradients for LogisticRegression only implemented without 'normalize'.")
 
     if base_algorithm.__class__.__name__ == 'SGDClassifier':
-        if base_algorithm.loss != 'log':
+        if base_algorithm.loss != 'log_loss':
```

Comment:
```suggestion
        if base_algorithm.loss not in ['log', 'log_loss']:
```

Thanks for the PR. Could you make it instead into an 'in' check accepting both the old 'log' and the new 'log_loss' from sklearn?